.github/workflows/release.yml

```diff
@@ -94,12 +94,26 @@ jobs:
           aws-region: ${{ env.BUILD_REGION }}
           role-to-assume: ${{ secrets.BUILD_ROLE_TO_ASSUME_PRODUCTION }}
           role-duration-seconds: 3600
+      # When called by Packer, Ansible will find /usr/bin/python3 and
+      # use it; therefore, we must ensure that /usr/bin/python3 points
+      # to the version of python that we installed.  This can hose
+      # other tasks that are expecting to find the system python at
+      # that location, though, so we undo this change after running
+      # packer.
```

Comment:
Same thoughts as above. I just wanted to mark it in both files.

Reply:
See commits fa6c1c1 and 51b729c.